lms/static/js/views/fields.js

```diff
@@ -100,24 +129,24 @@
 
             showNotificationMessage: function(message) {
                 this.$('.u-field-message-help').html('');
-                this.$('.u-field-message-notification').html(message);
+                HtmlUtils.setHtml(this.$('.u-field-message-notification'), message);
             },
 
             showCanEditMessage: function(show) {
                 if (!_.isUndefined(show) && show) {
-                    this.showNotificationMessage(this.getMessage('canEdit'));
+                    this.showNotificationMessage(this.getMessageHTML('canEdit'));
```

Comment:
1. I don't know if I'd rename `getMessage` to `getMessageHTML` without renaming all the others.  I don't like the idea of having 2 methods, `getMessage` and `getMessageHTML`.
2. If we did keep `getMessageHTML`, it should be `getMessageHtml`.